codec/address/bech32_codec.go

```diff
@@ -55,5 +55,20 @@ func (bc Bech32Codec) BytesToString(bz []byte) (string, error) {
 		return "", err
 	}
 
+	if err = verifyAddress(bz); err != nil {
```

Comment:
any reason to add a check here?

Reply:
As discussed on slack, we should remove the redundant 0 length check, but it makes sense to keep the max length check. We should probably as well move it up.